src/finch/__init__.py

```diff
@@ -100,8 +112,15 @@
     complex64,
     complex128,
     bool,
+    finfo,
+    iinfo,
 )
 
+logical_and = bitwise_and
+logical_or = bitwise_or
+logical_xor = bitwise_xor
+logical_not = bitwise_invert
```

Comment:
These are distinct -- `logical_*` casts to `bool` before any ops. `bitwise_*` performs bit-wise ops with integers.

Reply:
In Finch, logical and is the `Finch.and` function, while bitwise and is `&`

Reply:
I found:
```
Finch.and
Finch.or
Finch.xor
```
but I think `invert` or `not` is missing, right? (because it's tricky as it changes the fill value?)